server/eventlog/eventlog.go

```diff
@@ -143,18 +149,58 @@ func GetEventLogChunk(ctx context.Context, env environment.Env, req *elpb.GetEve
 }
 
 func NewEventLogWriter(ctx context.Context, b interfaces.Blobstore, invocationId string) *EventLogWriter {
+	cw := chunkstore.New(
+		b,
+		&chunkstore.ChunkstoreOptions{
+			WriteBlockSize:       defaultLogChunkSize,
+			WriteTimeoutDuration: defaultChunkTimeout,
+			NoSplitWrite:         true,
+		},
+	).Writer(ctx, getEventLogPathFromInvocationId(invocationId))
```

Comment:
nit: kinda messy / dense. Would be clearer if opts were defined before cw?

chunkstoreOptions := &chunkstore.ChunkstoreOptions{
			WriteBlockSize:       defaultLogChunkSize,
			WriteTimeoutDuration: defaultChunkTimeout,
			NoSplitWrite:         true,
		}
cw := chunkstore.New(b, chunkstoreOptions).Writer(ctx, getEventLogPathFromInvocationId(invocationId))